### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @lutonite

### DISCUSSION THREAD

**Discussion by Lutonite** [heig-vd-dai-course/heig-vd-dai-course#31] [Show and tell]
https://github.com/Lutonite/heig-dai-2023-pw

**Comment by ludelafo** [comment]
Thank you!

I would add a README to say a few words on what the project is about and how to build and run the app.

Also, in the `pom.xml`, you only have to keep one plugin (`maven-shade-plugin`), not both (`maven-jar-plugin` + `maven-shade-plugin`). We did update the documentation about this to avoid any confusion.

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
